data/wangsu

```diff
@@ -0,0 +1,4 @@
+8686c.com
```

Comment:
domains in this file is quite controvertible.
need more discussions before adding.

Reply:
`8686c.com` and `ourdvsss.com` are originally from `geolocation-cn`. They are CDNs of Wangsu used by Steam, JetBrains and Baidu.

Reply:
ok. understood.